CMakeLists.txt

```diff
@@ -1,25 +1,15 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
+cmake_policy(SET CMP0048 NEW)
```

Comment:
Is this necessary ? It looks like the default is NEW in CMake 3.5: https://cmake.org/cmake/help/v3.5/policy/CMP0048.html

Reply:
fc22ae0 

Oops, thanks. I think I set the policy first, then later upped the minimum required version and didn't check the policy's default.